wolfssl/wolfcrypt/types.h

```diff
@@ -917,6 +917,7 @@ typedef struct w64wrapper {
         WOLFSSL_API int wc_strncasecmp(const char *s1, const char *s2, size_t n);
     #endif
 
+    WOLFSSL_API char* wc_strdup(const char *src, int memType);
```

Comment:
This should not be public... Please make WOLFSSL_LOCAL

Reply:
Done